src/pyscipopt/scip.pxi

```diff
@@ -305,6 +310,19 @@ cdef class Event:
         """gets type of event"""
         return SCIPeventGetType(self.event)
 
+    def getName(self):
+        """gets name of event"""
+        if not EventNames:
+            self._getEventNames()
+        return EventNames[self.getType()]
+
+    def _getEventNames(self):
+        """gets event names"""
+        for name in dir(PY_SCIP_EVENTTYPE):
+            attr = getattr(PY_SCIP_EVENTTYPE, name)
+            if isinstance(attr, int):
+                EventNames[attr] = name
+        
     def __repr__(self):
         return self.getType()
```

Comment:
```suggestion
        return str(self.getType())
```
From what I understand __repr__ should always return a string so this behavior is weird.